packages/insomnia-app/app/ui/redux/selectors.ts

```diff
@@ -67,7 +68,7 @@ export const selectSpaces = createSelector(
 export const selectActiveSpace = createSelector(
   selectEntities,
   (state: RootState) => state.global.activeSpaceId,
-  (entities, activeSpaceId) => {
+  (entities, activeSpaceId): Space | undefined => {
     return entities.spaces[activeSpaceId || 'n/a'];
   },
```

Comment:
oh.  I see.  Is what's happening here is that we're trying to work around this error?

![image](https://user-images.githubusercontent.com/15232461/125998788-a9395ff2-d937-4c0e-ade4-5f4d20e6aeb0.png)

If so, I strongly feel that we should do this in a much less tricky way, for example:

```suggestion
  (entities, activeSpaceId) => {
    return activeSpaceId ? entities.spaces[activeSpaceId] : undefined;
  },
```

which matches the type you specified.

![image](https://user-images.githubusercontent.com/15232461/125998642-cbcccdfa-bf2b-4a00-8b0b-f813ff5a462c.png)

The reason I find it most tricky is that there are a good few places in the app which actually use this value, `'n/a'`.  If this isn't one of them (and, I'm just now realizing it probably is not one of them, although until now that's how I've been reading this line when I saw it before), I think we should change this.

Reply:
Great feedback and I agree; will improve this

Reply:
`'n/a'` is used in a bunch of different places but it doesn't mean anything, it may as well be `''`. Ideally we get rid of them but in some cases it's nice to have an actual meaningless value than nothing at all

Reply:
Fixed in [`3c614e6` (#3775)](https://github.com/Kong/insomnia/pull/3775/commits/3c614e6afb2d5317d90e53e62a6f451b1e565da0)